tools/@aws-cdk/pkglint/lib/rules.ts

```diff
@@ -166,6 +167,81 @@ export class LicenseFile extends ValidationRule {
   }
 }
 
+export class BundledCLI extends ValidationRule {
+
+  private static readonly VALID_LICENSES = [
+    'Apache-2.0',
+    'MIT',
+    'BSD-3-Clause',
+    'ISC',
+    'BSD-2-Clause',
+    '0BSD',
+  ];
+
+  private static readonly DONT_ATTRIBUTE = '^@aws-cdk\/|^cdk-assets$';
```

Comment:
The more I see those, the more I think we should actually model this as "first-party packages" (these don't need attribution since they belong to the same publisher / are part of the same project).

Note - we probably still need to attribute "our own" stuff that is from a different project/repository (although there is no such stuff here)...

Reply:
So `first-party` doesn't really fit..we can follow up on that at some point 👍